apps/vercel/frontend/src/constants/copies.ts

```diff
@@ -44,7 +44,8 @@ export const copies = {
         previewPath: {
           label: 'Preview path',
           placeholder: 'Set preview path and token',
-          errorMessage: 'This field is empty and not saved',
+          emptyErrorMessage: 'Field is empty and omitted from configuration',
+          invalidFormattingMessage: 'Formatting is invalid and omitted from configuration',
```

Comment:
Yeah I do think we need to tweak this message and presentation a bit. My thoughts:

* I think we should remove "omitted from configuration" entirely on both. Just dispaly the problem whatever it might be ("Field is empty" or "Formatting is invalid")
* Maybe display a warning instead of an error?

Reply:
The copies have been adjusted to be more succinct!